deluge/ui/web/server.py

```diff
@@ -740,6 +819,10 @@ def start(self):
 
         component.get('Web').enable()
 
+        # TODO: Check if correct
+        # Plugins must be enabled here to register exported json functions
+        self.plugins.enable_plugins()
```

Comment:
Were they not exported before?

Reply:
I think the problem I had was that the plugins list in the config was empty unless enabled here.